### ISSUE TITLE
Munge changes permissions of password file to 700

### ISSUE THREAD

**Issue by costrouc** [NixOS/nixpkgs#54947]
The munge module in nixos sets the permissions of the munge key to 700. Is this correct? the documentation on munge says to make it 0400. https://github.com/dun/munge/wiki/Installation-Guide#creating-a-secret-key

https://github.com/NixOS/nixpkgs/blob/0a5b4fda631ff1a950a0fc47c9ceab5c0da808c2/nixos/modules/services/security/munge.nix#L53

CC @markuskowa

**Comment by markuskowa** [comment]
You are right. It should not be 0700. At least the executable bit makes no sense here. We can change it to the recommendation 0400.

**Comment by costrouc** [comment]
I will open a PR today to fix the issue
